.github/workflows/sidecar-check.yml

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2020-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
```

Comment:
please add a description/purpose for this script so future maintainers will know why we're doing this